slo_generator/report.py

```diff
@@ -42,22 +42,11 @@ class SLOReport:
     """
     # pylint: disable=too-many-instance-attributes
 
-    # Metadata
-    metadata: dict = field(default_factory=dict)
-
     # SLO
     name: str
     description: str
     goal: str
     backend: str
-    exporters: list = field(default_factory=list)
-    error_budget_policy: str = 'default'
-
-    # SLI
-    sli_measurement: float = 0
-    events_count: int = 0
-    bad_events_count: int = 0
-    good_events_count: int = 0
     gap: float
```

Comment:
Add an empty line and a `# SLI` comment above this line to clearly indicate this variable is related to an SLI. Otherwise we are under the impression it belongs to the SLO section now that the variable definitions have moved elsewhere.